components/Storyblok/SbStoryMvp/SbStoryMvp.tsx

```diff
@@ -69,71 +58,34 @@ export const SbStoryMvp = ({
       <Masthead isLight={isLightHero} />
       <main id="main-content">
         <Container as="article" width="full" pb={9}>
-          {!(title?.includes('Whereas') || title?.includes('Progress') || title?.includes('Video') || title?.includes('Solve')) && (
-            <StoryHeroMvp
-              title={title}
-              customHeading={customHeading}
-              headingFont={headingFont}
-              isSmallHeading={isSmallHeading}
-              dek={dek}
-              byline={byline}
-              publishedDate={publishedDate}
-              heroImage={heroImage}
-              aspectRatio={aspectRatio}
-              mobileImage={mobileImage}
-              mobileAspectRatio={mobileAspectRatio}
-              bgImage={bgImage}
-              addBgBlur={addBgBlur}
-              addDarkOverlay={addDarkOverlay}
-              isLeftImage={isLeftImage}
-              alt={alt}
-              caption={caption}
-              isLightHero={isLightHero}
-              isVerticalHero={isVerticalHero}
-              tabColor={tabColor}
-              topics={topics}
-            />
-          )}
+          <StoryHeroMvp
+            title={title}
+            customHeading={customHeading}
+            headingFont={headingFont}
+            isSmallHeading={isSmallHeading}
+            dek={dek}
+            byline={byline}
+            publishedDate={publishedDate}
+            heroImage={heroImage}
+            aspectRatio={aspectRatio}
+            mobileImage={mobileImage}
+            mobileAspectRatio={mobileAspectRatio}
+            bgImage={bgImage}
+            addBgBlur={addBgBlur}
+            addDarkOverlay={addDarkOverlay}
+            isLeftImage={isLeftImage}
+            alt={alt}
+            caption={caption}
+            isLightHero={isLightHero}
+            isVerticalHero={isVerticalHero}
+            tabColor={tabColor}
+            topics={topics}
+          />
           {showAboveContent && (
             <SbAboveContent aboveSidebar={aboveSidebar} intro={intro} sidebar={sidebar} />
           )}
-          {title?.includes('Solve') && (
-            <BrochureStory />
-          )}
-          {title?.includes('Progress') && (
-            <ProgressStory />
-          )}
-          {title?.includes('Video scrolling') && (
-            <VideoScrollStory />
-          )}
-          {title?.includes('Chatbot') && (
-            <ChatbotStory />
-          )}
-          {title?.includes('Whereas') && (
-            <MulticolumnStory />
-          )}
```

Comment:
Removed a bunch of stuff (I cloned the original SbStory component when I created the SbStoryMvp component). There are only used for SbStory (some POCs), but not for this Mvp one.